docs/notebook/Tutorial.ipynb

```diff
@@ -757,6 +757,40 @@
     "print(conf.key)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Nested interpolations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can use interpolations within interpolations, for instance to perform an\n",
+    "operation over config variables:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "3\n"
+     ]
+    }
+   ],
+   "source": [
+    "OmegaConf.register_resolver(\"plus_int\", lambda x, y: int(x) + int(y))\n",
+    "conf = OmegaConf.create({\"a\": 1, \"b\": 2, \"a_plus_b\": \"${plus_int:${a},${b}}\"})\n",
+    "print(conf.a_plus_b)"
+   ]
+  },
```

Comment:
Can you also add an example for a simple nested interpolation?
something along the lines of:
```python
cfg = OmegaConf.create({
  "plans":{
    "A": "plan A",
    "B": "plan B",
  },
  "selected_plan": "A"
  "plan": "${plans.${selected_plan}}"
})

cfg.plan
>> "Plan A"
cfg.selected_plan = "B"
cfg.plan
>> "Plan B"
```

Reply:
Good idea! Done in a15623d